phoenix/src/main.rs

```diff
@@ -261,19 +270,25 @@ mod app {
         sbg_power.set_high();
 
         // UART for sbg
-        let tx: Pin<'D', 1, Alternate<8>> = gpiod.pd1.into_alternate();
-        let rx: Pin<'D', 0, Alternate<8>> = gpiod.pd0.into_alternate();
-
-        // let stream_tuple = StreamsTuple::new(ctx.device.DMA1, ccdr.peripheral.DMA1);
+        let tx: Pin<'A', 0, Alternate<8>> = gpioa.pa0.into_alternate();
+        let rx: Pin<'A', 1, Alternate<8>> = gpioa.pa1.into_alternate();
+        let stream_tuple = StreamsTuple::new(ctx.device.DMA1, ccdr.peripheral.DMA1);
+        let uart_sbg = ctx
+            .device
+            .UART4
+            .serial((tx, rx), 57600.bps(), ccdr.peripheral.UART4, &ccdr.clocks)
+            .unwrap();
+        let sbg_manager = sbg_manager::SBGManager::new(uart_sbg, stream_tuple);
+        
+        // UART for radio
+        let tx: Pin<'E', 8, Alternate<8>> = gpioe.pe8.into_alternate();
+        let rx: Pin<'E', 7, Alternate<8>> = gpioe.pe7.into_alternate();
```

Comment:
These pins need to be corrected. I think the sbg one is correct per Altium, but the radio is an issue.